### ISSUE TITLE
Add Heights endpoints

### ISSUE THREAD

**Issue by RishavOsaurus** [RishavOsaurus/NepalMoutainsAPI#3]
Make search like search?height=8848.68

GET /api/v1/search?height_gt=1230.4&height_lt=4000
GET /api/v1/search?height_gt=1230.4
GET /api/v1/search?height_st=1230.4

**Comment by Indrajeet-S** [comment]
Hey, I would like to add this heights endpoint feature! Could you assign this issue to me?

**Comment by RishavOsaurus** [comment]
Hi I have assigned it to you @Indrajeet-S

**Comment by Indrajeet-S** [comment]
Thanks but I can't see in `assignees` nvm I will start working on this issue!

**Comment by RishavOsaurus** [comment]
My bad! Just assigned now

**Comment by shinigami-777** [comment]
@RishavOsaurus Can I work on this too?

**Comment by RishavOsaurus** [comment]
> @RishavOsaurus Can I work on this too?

yes you can but better you two guys coooperate and do work distribution so that you dont duplicate the work already done

**Comment by shinigami-777** [comment]
@RishavOsaurus Can you review this PR. I have tested it out locally with the added search parameters.
